pages/index.js

```diff
@@ -39,14 +42,37 @@ export default function HomePage({
     setModalFilter(!modalFilter);
   }
 
+  function showSearchField() {
+    setIsHidden(!isHidden);
+  }
+
+  const fuse = new Fuse(projects, {
+    keys: ["title", "description", "materials", "steps.desc"],
+    includeScore: true,
+    threshold: 0.3,
+    shouldSort: true,
+    ignoreLocation: true,
+    ignoreFieldNorm: true,
+  });
+  const results = fuse.search(query);
+  console.log("results: ", results);
```

Comment:
Whilst i am a big fan of logging during development, I don't think we want a console log making it into production.